lambdas/src/common/config/config-service.ts

```diff
@@ -76,22 +76,20 @@ export class ConfigService {
     }
 
     public getAuthorizationCodeExpirationEpoch() {
-        // TODO: consider if this should be output in epoch seconds rather than milliseconds
-        // so that it is consistent with the java implementation
-        return Date.now() + this.authorizationCodeTtlInMillis;
+        return Math.floor(Date.now() + this.authorizationCodeTtlInMillis) / 1000;
```

Comment:
I think the `Math.floor()` should apply to the whole phrase - if we're doing the floor before the /1000 we'll still get decimals that we don't want

Reply:
Good spot - updated